index.d.ts

```diff
@@ -7,6 +7,10 @@ import {
   StoreEnhancer
 } from 'redux';
 
+
+import {History} from "history";
+import React from "react";
```

Comment:
neither is unnecessary.
line 11: @types/react-router has dependency on @types/history, not the one in lib.es6.d.ts.
line 12 imports JSX namespace referenced in line 80.